### ISSUE TITLE
Bug related to the texture of some items

### ISSUE THREAD

**Issue by Quaerere-Git** [GregTech6/gregtech6#65]
Some items looks different when they're in the inventory, dropped on the ground and held in hand.
![2022-01-12_21 03 24](https://user-images.githubusercontent.com/42565872/149145869-319c99f8-12f7-416e-bc5b-8a52b70a175f.png)Stainless Steel ingot show wrong texture in some cases
![2022-01-12_21 04 29](https://user-images.githubusercontent.com/42565872/149145877-d19fb909-d2bc-4b59-9b76-9f2600f395fe.png)A len with color lost color in some cases

**Comment by Quaerere-Git** [comment]
Some additional information:

OS: Windows 10.0 , 64 bit Build 22523 (10.0.22523.1000)

CPU:total 8 (initial active 8) (4 cores per cpu, 2 threads per core) family 6 model 165 stepping 2, cmov, cx8, fxsr, mmx, sse, sse2, sse3, ssse3, sse4.1, sse4.2, popcnt, avx, avx2, aes, clmul, erms, 3dnowpref, lzcnt, ht, tsc, tscinvbit, bmi1, bmi2, adx

Memory: 4k page, physical 16609072k(5492100k free), swap 19512352k(3180412k free)

vm_info: Java HotSpot(TM) 64-Bit Server VM (25.291-b10) for windows-amd64 JRE (1.8.0_291-b10), built on Apr  9 2021 00:02:00 by "java_re" with MS VC++ 15.9 (VS2017)

**Comment by GregoriusT** [comment]
Yep, I dont really know how to rewrite that vanilla Renderer to make transparency actually work. That is all that one is about. :/
